RIAssigner/data/MatchMSData.py

```diff
@@ -91,7 +91,9 @@ def _spectrum_has_rt(spectrum: Spectrum) -> bool:
     has_key = 'retentiontime' in spectrum.metadata.keys()
```

Comment:
You are basically checking the presence of `'retentiontime'` twice. Could be simplified by only using `safe_read_key` function.

Reply:
I removed the function as it is no longer required.